src/compute.rs

```diff
@@ -178,3 +179,10 @@ pub fn merkle_root(data: &[u8]) -> [u8; 32] {
 
     hasher.finalize().into()
 }
+
+/// Get a Point from an ecdsa::PublicKey
+pub fn point(key: &ecdsa::PublicKey) -> Result<Point, PointError> {
+    let compressed = Compressed::from(key.to_bytes());
+    // this should not fail as long as the public key above was valid
+    Point::try_from(&compressed)
+}
```

Comment:
Why isn't there an infallible way to go from a public key to a point? One is supposed to be superset of the other.

Reply:
Previously we just called `unwrap` since as you note it should not fail.  But it seemed cleaner to remove the `unwrap` call and chain up the error, even though it should never be triggered.  

I'm kind of on the fence here.  Previous versions of `p256k1` made the internal `secp256k1_pubkey` struct `pub`, so it was possible for the internal data to be manipulated in such a way that the `unwrap` could be triggered.  The current `master` branch has moved this to `pub(crate)`, which reduces the ability for this to happen, but that code hasn't been released.

Do you think this is worth blocking this change?

Reply:
Oh right, I see. Okay all good here then.

Not relevant for this PR, but any reason why we shouldn't add an implementation for infallibly converting a `PublicKey` to a `Point` into `p256k1` and bumping the dependency version here?

Reply:
No reason not to.  I wanted to get this PR in first though since Adriano wants to release the audit report.